zebra/zebra_vxlan.c

```diff
@@ -60,6 +60,10 @@ DEFINE_MTYPE_STATIC(ZEBRA, MAC, "VNI MAC");
 DEFINE_MTYPE_STATIC(ZEBRA, NEIGH, "VNI Neighbor");
 DEFINE_MTYPE_STATIC(ZEBRA, ZVXLAN_SG, "zebra VxLAN multicast group");
 
+DEFINE_HOOK(zebra_rmac_update, (zebra_mac_t *rmac, zebra_l3vni_t *zl3vni,
+		bool delete, const char *reason),
```

Comment:
You need to fix your tab/spacing issues.  There are multiple places throughout the whole commit that need this.